storage_service/common/startup.py

```diff
@@ -149,10 +140,10 @@ def populate_default_locations():
         if created and loc_info.get("create_dirs"):
             LOGGER.info("Creating %s Location %s", loc_info["purpose"], new_loc)
             try:
-                os.mkdir(new_loc.full_path)
+                pathlib.Path(new_loc.full_path).mkdir()
                 # Hack for extra recovery dir
                 if loc_info["purpose"] == locations_models.Location.AIP_RECOVERY:
-                    os.mkdir(os.path.join(new_loc.full_path, "backup"))
+                    pathlib.Path(new_loc.full_path).joinpath("backup").mkdir()
```

Comment:
Could these two `pathlib.Path(new_loc.full_path)` be extracted to a variable above?

I think that would also allow you to do `(NEW_VAR / "backup").mkdir()`.

Reply:
Done, the requested change has been implemented and pushed to the branch.